### PR TITLE
CMake: set default RUNTIME_OUTPUT_DIRECTORY property to "bin"

### PR DESCRIPTION
**Pull request by chfast**
This commit changes output directory for runtime components (executables and DLLs) to "bin" directory. That allows running executables on Windows without need of install step.

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -95,6 +95,12 @@ if (JSONRPC)
 	target_link_libraries(testeth ${JSON_RPC_CPP_CLIENT_LIBRARIES})
 endif()
 
+if (UNIX) # Create symlink to old testeth location to make bildbot happy
```
Comment (debris) [inline test/CMakeLists.txt]: :+1:

**Comment by chfast** [comment]
Let wait for test builds...

**Comment by debris** [comment]
sure :)

**Comment by chfast** [comment]
As it was reverted... should we continue working on that?
I used a symlink test/testeth -> bin/testeth to keep buildbot testing working. We can try the same for other case. But I'm not sure it is worth our time.

**Pull request merged**